components/script/dom/virtualmethods.rs

```diff
@@ -124,9 +124,9 @@ pub(crate) trait VirtualMethods {
     }
 
     /// Called on the parent when its children are changed.
-    fn children_changed(&self, mutation: &ChildrenMutation, _can_gc: CanGc) {
+    fn children_changed(&self, mutation: &ChildrenMutation, can_gc: CanGc) {
```

Comment:
I named it `_can_gc` in https://github.com/servo/servo/pull/40325 because I thought clippy would be mad otherwise, but turns out I was wrong.